code/modules/clothing/head/f13head.dm

```diff
@@ -222,13 +222,20 @@
 	dynamic_hair_suffix = ""
 	dynamic_fhair_suffix = ""
 	speechspan = SPAN_ROBOT //makes you sound like a robot
+	heat_protection = HEAD
+	max_heat_protection_temperature = FIRE_HELM_MAX_TEMP_PROTECT
+	cold_protection = HEAD
+	min_cold_protection_temperature = FIRE_HELM_MIN_TEMP_PROTECT
 //	darkness_view = 128
 //	lighting_alpha = LIGHTING_PLANE_ALPHA_NV_TRAIT
 	var/on = FALSE
 	var/brightness_on = 5
 	var/power_on = 1
 	var/emped = 0
 	var/requires_training = TRUE
+	var/armor_block_chance = 0
```

Comment:
If you want to do the sane thing and change it to checking armour penetration on things it's hit with (as the var is available on anything derived from `/obj/item`), this should probably be replaced with
```suggestion
	var/armour_block_threshold = 0.75
```
or something similar.
Note that you should probably use a spreadsheet of AP values for this in order to fine-tune what guns hit what armours. It should feel much more natural if any extra ammo types are added, *and* it should also handle things that aren't subtypes of projectiles, i.e. a scrap launcher or something.